cirq/devices/grid_qubit.py

```diff
@@ -24,6 +24,11 @@ class GridQubit(ops.Qid):
     GridQubits use row-major ordering:
 
         GridQubit(0, 0) < GridQubit(0, 1) < GridQubit(1, 0) < GridQubit(1, 1)
+
+    New GridQubits can be constructed by adding or subtracting tuples
+
+        GridQubit(2, 3) + (3, 1) produces GridQubit(5, 4)
+        GridQubit(2, 3) - (1, 2) produces GridQubit(1, 1)
```

Comment:
Format it like
```python
>>> GridQubit(2, 3) + (3, 1)
GridQubit(5, 4)
```
so the new doctest can test it.

Reply:
Done.  It ends up being
```
>>> GridQubit(2, 3) + (3, 1)
cirq.GridQubit(5, 4)
```
It might be nice to import cirq into doc test so this could be
```
>>> cirq.GridQubit(2, 3) + (3, 1)
cirq.GridQubit(5, 4)
```